buildpack.md

```diff
@@ -1138,6 +1145,41 @@ Each `key`:
 ## Deprecations
 This section describes all the features that are deprecated.
 
+### buildpack.toml (TOML) `stacks` Array
+
+_Deprecated in Buildpack API 0.10._
+
+The `stacks` array is deprecated.
+
+```toml
+[[stacks]]
+id = "<stack ID>"
+mixins = ["<mixin name>"]
+```
+
+Each stack in `stacks` either:
```

Comment:
I'm too lazy to find the other PR, but fwiw i think in this one you _did_ move all the deprecated stack specifications down into this deprecated section.